src/frontends/paddle/src/internal/pass/transform_fakequantize.cpp

```diff
@@ -0,0 +1,132 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "internal/pass/transform_fakequantize.hpp"
+
+#include <ngraph/ngraph.hpp>
+#include <ngraph/op/util/op_types.hpp>
+#include <ngraph/pattern/matcher.hpp>
+#include <ngraph/pattern/op/or.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/rt_info.hpp>
+
+#include "default_opset.hpp"
+#include "openvino/pass/pattern/op/label.hpp"
+#include "transformations/utils/utils.hpp"
+
+using namespace ov::frontend::paddle::op::default_opset;
+using namespace ov;
+using namespace ov::pass;
+using namespace ov::frontend::paddle::op;
+
+/*
+                                  zero_point
+                                      /
+                        input    convert     scale
+                           \        /         /
+                            subtract  Multiply
+   quantize_linear   ==>>        \      /
+                                  Divide
+                                     \
+                                      Round
+                                         \
+                                          Clamp
+                                             \
+  _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _   => FakeQuantize
+                                               \           zero_point
+                                                \              /
+                                                Convert   Convert   scale
+   dequantize_linear  ==>>                          \      /        /
+                                                   Subtract    Multiply
+                                                       \        /
+                                                        Multiply
+*/
+ov::frontend::paddle::pass::TransformFakeQuantize::TransformFakeQuantize() {
+    const auto input_label = ngraph::pattern::any_input();
+    const auto q_zp_label = ngraph::pattern::any_input();
+    // quantize phase
+    const auto q_zp_cvt_label = ngraph::pattern::wrap_type<Convert>({q_zp_label});
+    const auto q_sub_label = ngraph::pattern::wrap_type<Subtract>({input_label, q_zp_cvt_label});
+    const auto q_real_scale_label = ngraph::pattern::wrap_type<Multiply>();
+    const auto div_label = ngraph::pattern::wrap_type<Divide>({q_sub_label, q_real_scale_label});
+    const auto round_label = ngraph::pattern::wrap_type<Round>({div_label});
```

Comment:
@xczhai, you said you agreed to ignore rounding mode, but we still generating this `Round` in quantize_linear op translation and catch it here to get rid of it. This is useless if you really can ignore it. "Ignore" means to convert it to FQ anyway regardless of real rounding used. It probably provides a bit lower accuracy -- but could you provide at least one example when it really does? Instead it will give you always good performance because you are triggering real int8 execution in this way.